internal/wrphandlers/mocktr181/handler.go

```diff
@@ -181,20 +184,34 @@ func (h Handler) get(names []string) (int64, []byte, error) {
 	return statusCode, payload, nil
 }
 
-func (h Handler) set(parameters []Parameter) int64 {
+func (h Handler) set(parameters []Parameter) (int64, []byte, error) {
+	result := Tr181Payload{}
 	for _, parameter := range parameters {
-		for _, mockParameter := range h.parameters {
+		for i := range h.parameters {
+			mockParameter := &h.parameters[i]
 			if strings.HasPrefix(mockParameter.Name, parameter.Name) {
 				if mockParameter.Access == "rw" {
 					mockParameter.Value = parameter.Value
 					mockParameter.DataType = parameter.DataType
 					mockParameter.Attributes = parameter.Attributes
+
+					result.Parameters = append(result.Parameters, Parameter{
+						Name:       mockParameter.Name,
+						Value:      mockParameter.Value,
+						DataType:   mockParameter.DataType,
+						Attributes: mockParameter.Attributes,
+					})
```

Comment:
@ilawjr asked for mock181 SET's response payload to contain the update parameters